src/locales/messages.pot

```diff
@@ -3200,6 +3203,9 @@ msgstr ""
 msgid "Transfer {tokenTextShort}"
 msgstr ""
 
+msgid "Transfers are paused"
```

Comment:
Transfers of project tokens are paused. This does not apply to ERC-20 tokens if issued.